### ISSUE TITLE
Add Cisco support

### ISSUE THREAD

**Issue by enduring78** [eduardogsilva/routerfleet#12]
Cisco switches have built-in commands for showing the config (within Enabled mode, run "show running config'' ). 

Since they also have SSH and Telnet support, it might be a good brand to start supporting alongside MikroTik. 
I would imagine only showing the text version of the backup, without binary download option. Cisco devices are entirely based on shell command so it would be easy to just copy/paste the backup contents and overwrite the existing config.

**Comment by eduardogsilva** [comment]
I do believe that ciscos are a good choice to add. 
I just need to fix some papercuts before starting to add routers. 
It will be soon. Maybe a week or two.

**Comment by hellofaduck** [comment]
I apologize for not posting in a new thread, but it would be good to add support for SwOS, binary backups from there can be retrieved via http with basic auth, should not be difficult to implement. This would allow to cover all mikrotik devices, not only RouterOS. Thanks for your work, this is a very useful project!

**Comment by eduardogsilva** [comment]
> I apologize for not posting in a new thread, but it would be good to add support for SwOS, binary backups from there can be retrieved via http with basic auth, should not be difficult to implement. This would allow to cover all mikrotik devices, not only RouterOS. Thanks for your work, this is a very useful project!
Hi, please open a new issue so it can be discussed there.
This issue is for cisco support.
